.github/workflows/ios.yml

```diff
@@ -0,0 +1,38 @@
+name: iOS PR CI
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+  test:
+    name: Preflight
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache SPM packages
+        uses: actions/cache@v2
+        with:
+          path: /Users/runner/Library/Developer/Xcode/DerivedData/**/SourcePackages/checkouts
+          key: ${{ runner.os }}-spm-${{ hashFiles('**/Package.resolved') }}
+          restore-keys: |
+            ${{ runner.os }}-spm-
+
+      - name: Lint code
+        run: make lint
+
+      - name: Run unit tests
+        run: make test
+
+      - name: Build preview apps
+        run: make build_preview_apps
+
+      - name: Build release scheme
+        run: release_build
```

Comment:
Shouldn't this be `make release_build`?

Reply:
Thanks. I didn't request a review even though you're right. But as you can see the workflow isn't even running yet.

Reply:
No problem, thanks for the PR, I was happy to see this 🙂